bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx133-donation-eu-na.html

```diff
@@ -0,0 +1,12 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% extends "firefox/whatsnew/whatsnew-fx133-donation.html" %}
+
+{% block experiments %}
```

Comment:
**Issue:** if there's no experiment file in this template then it can't send the right GA event for v2 or v3?

Might it make sense to have a separate template for EU and NA here, each extending `whatsnew-fx133-donation.html` and then loading it's required experiment file?